packages/api-client/src/components/ApiClient/RequestHistory.vue

```diff
@@ -40,8 +40,238 @@ const { requestHistoryOrder } = useRequestStore()
     </div>
   </div>
 </template>
-
 <style>
+.navtable {
```

Comment:
Some of these un scoped styles I just moved to be closer to the components that consume them.